src/nat/eval/runners/red_teaming_runner/__init__.py

```
@@ -13,12 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from nat.eval.runners.red_teaming_runner.runner import RedTeamingRunner
-from nat.eval.runners.red_teaming_runner.config import RedTeamingRunnerConfig
-from nat.eval.runners.red_teaming_runner.config import RedTeamingScenario
+from nat.eval.runners.red_teaming_runner.config import RedTeamingRunnerConfig  # noqa: F401
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)